.gitignore

```diff
@@ -1,3 +1,4 @@
+.DS_Store
```

Comment:
I recommend adding this to your global gitignores, as it is not specific to this repository.